src/lib/core/components/Form/types/array.ts

```diff
@@ -16,13 +16,30 @@ import {
     ValidatorsMap,
 } from './';
 
-export type ArrayInputProps = InputProps<FieldArrayValue, ArraySpec>;
-export type ArrayIndependentInputProps = IndependentInputProps<FieldArrayValue, ArraySpec>;
-export type ArrayLayoutProps = LayoutProps<FieldArrayValue, ArraySpec>;
+export type ArrayInputProps<IProps extends Record<string, any> = {}> = InputProps<
+    FieldArrayValue,
+    ArraySpec<any, IProps>
```

Comment:
here and in other cases, it will always be undefined, not any

Reply:
but default is `any`..